### DISCUSSION TITLE
You broke @ant-design/charts for months ...

### DISCUSSION THREAD

**Discussion by azro352** [ant-design/ant-design#38920] [Q&A] [answered]
Hi
Just tried to update to antd@5.0. 0 in a project who uses ant-design/charts too and after some research I've discovered that I won't be able to really use 5.0.0 for the next month : 

Due to CSS-in-JS:
1. [x6-react-components](https://github.com/antvis/X6/tree/master/packages/x6-react-components) is broken, if resolved then :
2. [antv/xflow](https://github.com/antvis/XFlow) who have low activity would need to be update too
3. [@ant-design/flowchart](https://github.com/ant-design/ant-design-charts/tree/master/packages/flowchart) would then be needed to be updated, regarding issues regarding package upgrade, the project seems to be in low activity too

So I'm asking : what to do ? It seems to be a long journey before being able to use antd5.0.0 along with antd/charts
I'm ready to make PRs in the projects but that seems a big task

Thanks !

**Comment by zombieJ** [comment]
Please ref to follow `package.json` `peerDependencies`. It's still v4:
* https://github.com/antvis/X6/blob/e0e7e2d80f5cacc8b585de49804c96f46b97df52/packages/x6-react-components/package.json#L51
* https://github.com/ant-design/ant-design-charts/blob/83f1ba7a3328a6d0d499e7778317c1c73938293a/packages/flowchart/package.json#L51

cc @neoddish

**Reply by azro352** [reply]
I know 🤔 

What I'm saying is that x6-react-components uses an .css import, so i just can't upgrade to antd v5 in my project that has ant-design/charts too, because it fails (see https://github.com/antvis/X6/pull/2921 and related issue in it). So I said, x6-r-c needs to be updated, then xflow (that uses x6-r-c), then charts that uses both x6 directly and xflow

**Comment by lxfu1** [comment]
Recent support.

**Comment by lxfu1** [comment, marked as answer]
https://github.com/ant-design/ant-design-charts/issues/1689
